docs/specifications/tests/Connectivity-TP/connectivity02.md

```diff
@@ -1,48 +1,196 @@
-# CONNECTIVITY02: TCP connectivity
+# CONNECTIVITY02: TCP connectivity to name servers
 
 ## Test case identifier
-
 **CONNECTIVITY02**
 
+
+## Table of contents
+
+* [Objective](#objective)
+* [Scope](#scope)
+* [Inputs](#inputs)
+* [Summary](#summary)
+* [Test procedure](#test-procedure)
+* [Outcome(s)](#outcomes)
+* [Special procedural requirements](#special-procedural-requirements)
+* [Intercase dependencies](#intercase-dependencies)
+* [Terminology](#terminology)
+
+
 ## Objective
 
-DNS queries are sent using TCP on port 53, as described in
-[RFC 1035][RFC 1035#section-4.2.2], section 4.2.2. As specified in
-[RFC 7766][RFC 7766#section-1], section 1, support of TCP is mandatory.
+TCP is a protocol to reach a general purpose name server hosting a zone, "All
+general-purpose DNS implementations MUST support [...] TCP transport"
+([RFC 7766][RFC 7766#5], section 5).
 
-The objective for this test is that all the authoritative name servers for
-the domain are accessible over TCP on port 53.
+This test case will verify if the name servers in the delegation of and zone of
+*Child Zone* are reachable over TCP.
+
+This test case will mimic the tests done by [Connectivity01], but over TCP
+instead:
+
+* Name Server responding to a query.
+* Name Server including SOA record of *Child Zone* in the answer section in the
+  response on a SOA query for *Child Zone*.
+* Name Server including NS record of *Child Zone* in the answer section in the
+  response on an NS query for *Child Zone*.
+* Name Server setting the AA flag in a response with SOA or NS in answer section.
+* Name Server responding with expected [RCODE Name] ("NoError") on query for SOA
+  or NS for *Child Zone*.
 
-## Inputs
 
-The domain to be tested.
+## Scope
 
-## Ordered description of steps to be taken to execute the test case
+The only TCP port defined for DNS is port 53 ([RFC 1035][RFC 1035#4.2.1], section
+4.2.1), and that is the only port used by this and other test cases for DNS
+queries to the name servers.
+
+UPD connectivity is tested by test case [Connectivity01].
+
+
+## Inputs
+
+* "Child Zone" - The domain name to be tested.
+
+
+## Summary
+
+Message Tag                         |Level  |Arguments  | Message ID for message tag
+:-----------------------------------|:------|:----------|:---------------------------------------------------------------------------------------
```

Comment:
Those were left out from this Test Case on purpose. If included here it would just result in the messages being repeated as they are outputted by Connectivity01. Reducing repeated messages not giving new information is a goal.

The messages of disabled IPv4 or IPv6 is not a strict test case message, but rather some kind of system message. At the same time it is good to report what name servers that will not be tested. Should there instead be a special Basic Test Case that reports this?